components/datetime/tests/fixtures/tests/components.json

```diff
@@ -25,6 +25,9 @@
                 "ja-u-ca-japanese": "令和2年1月21日火曜日 8:25:07",
                 "en-u-ca-coptic": "Tuesday, Toba 12, 1736 ERA1, 08:25:07",
                 "fr-u-ca-coptic": "mardi 12 toubah 1736 après Dioclétien 08:25:07",
+                "en-u-ca-dangi": "Tuesday, Twelfth Month 27, 2019((cyclic year: 36)), 08:25:07",
+                "fr-u-ca-dangi": "2019((cyclic year: 36)) shí’èryuè 27, mardi 08:25:07",
+                "ko-u-ca-dangi": "2019년((cyclic year: 36)년) 12월 27일 화요일 08:25:07",
```

Comment:
observation: the double parentheses are because we're using parentheses as fake formatting patterns until https://github.com/unicode-org/icu4x/issues/3761 is fixed